ethereum/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -148,7 +189,40 @@ contract L1ERC20Bridge is IL1Bridge, IL1BridgeLegacy, AllowListed, ReentrancyGua
         uint256 _l2TxGasLimit,
         uint256 _l2TxGasPerPubdataByte
     ) external payable returns (bytes32 l2TxHash) {
-        l2TxHash = deposit(_l2Receiver, _l1Token, _amount, _l2TxGasLimit, _l2TxGasPerPubdataByte, address(0));
+        // KL todo 9, get default Era chainId
+        l2TxHash = deposit(9, _l2Receiver, _l1Token, _amount, _l2TxGasLimit, _l2TxGasPerPubdataByte, address(0));
```

Comment:
This and the function bellow have different chainIDs, just in case, chainID should not only be stored in the constant but also used in both functions